### DISCUSSION TITLE
nnUNetv2 train Error

### DISCUSSION THREAD

**Discussion by marufmore** [MIC-DKFZ/nnUNet#1454] [General]
I get the following error

   assert not self._wait_semaphore.acquire(
OSError: [WinError 6] The handle is invalid

when I run this

nnUNetv2_train 137 3d_fullres 4 --npz

nnUNetv2_plan_and_preprocess -d 137 --verify_dataset_integrity  #this code runs fine and it created the required folders

**Comment by ancestor-mithril** [comment]
nnUNet does not have official support for Windows. 
You may try modifying the entrypoint for nnUNetv2_train (https://github.com/MIC-DKFZ/nnUNet/blob/master/nnunetv2/run/run_training.py) according to this: https://docs.python.org/2/library/multiprocessing.html#windows.

**Comment by marufmore** [comment]
I effected the above and got this error next

return F.conv3d(
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 18.00 MiB (GPU 0; 2.00 GiB total capacity; 1.65 GiB already allocated; 0 bytes free; 1.66 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF
Exception in thread Thread-4 (results_loop):

**Reply by ancestor-mithril** [reply]
Please check https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/installation_instructions.md#hardware-requirements-for-training.
